frontend/src/components/models.ts

```diff
@@ -13,9 +13,47 @@ export interface MulticallResponse {
   returnData: string[];
 }
 
+export type Chain = {
+  chainId: string;
+  chainName: string;
+  nativeCurrency: {
+    name: string;
+    symbol: string;
+    decimals: 18;
+  };
+  rpcUrls: string[];
+  blockExplorerUrls?: string[];
+  iconUrls?: string[];
+};
+export const supportedChains: Array<Chain> = [
```

Comment:
I like this layout for defining support chains. A few comments though:
- Can we make `rpcUrls`, `blockExplorerUrls`, and `iconUrls` from plural/arrays into singular/strings? Not sure I see the reasoning for making them arrays at the moment
- Can we add a `logoURI: string;` field to the `nativeCurrency` object? This way we have the token symbol of the native currency also. For ETH we can use `'/tokens/eth.svg'` as the URI

Reply:
I took this interface out of [EIP](https://eips.ethereum.org/EIPS/eip-3085#specification).

Reply:
Oh! Didn't realize that. Ok, fine with me to leave this as-is then 👌

Reply:
Maybe throw in a comment with a link to that EIP if you don't mind!